etc/rita.yaml

```diff
@@ -89,3 +89,23 @@ Beacon:
     # increase this value to improve performance if you are not concerned
     # about slow beacons.
     DefaultConnectionThresh: 20
+
+Filtering:
+    # These are filters that affect the import of connection logs.
+    #
+    # Example: AlwaysInclude: ["8.8.8.8/32"]
+    # This functionlity overrides the NeverInclude and InternalSubnets
+    # section, making sure that any connection records containing addresses from
+    # this range are kept and not filtered
+    #
+    # Example: NeverInclude: ["8.8.4.4/32"]
+    # This functions as a whitelisting setting, and connections involving
+    # ranges entered into this section are filtered out at import time
+    #
+    # Example: InternalSubnets: ["10.0.0.0/8","172.16.0.0/12","192.168.0.0/16"]
+    # This allows a user to identify their internal network, which will result
+    # in any internal to internal and external to external connections being
+    # filtered out at import time.
+    AlwaysInclude: []
+    NeverInclude: []
+    InternalSubnets: ["10.0.0.0/8","172.16.0.0/12","192.168.0.0/16"]
```

Comment:
I know we have a script that appends the `Filtering` block in later which is why it wasn't included in the first place.  But I just tested out what happens if there are two `Filtering` blocks and the one defined later (e.g. appended to the end) takes precedence. So it should be safe to leave these in the default config (just maybe a little confusing).

Or would it be better to comment all these out by default in order to preserve RITA's existing functionality with no internal networks defined? Since we're doing actual filtering based on these values and we can't know for sure which ones apply to a given user maybe it would be safer to just not do any filtering by default unless the user (or a script) purposely sets these. Thoughts?

Reply:
commented out